doc/man7/provider-asym_cipher.pod

```diff
@@ -193,10 +193,19 @@ further details.
 Gets or sets the name of the OAEP digest algorithm used when OAEP padding is in
 use.
 
+=item "digest" (B<OSSL_ASYM_CIPHER_PARAM_DIGEST>) <UTF8 string>
+
+Gets or sets the name of the digest algorithm used by the algorithm (where
+applicable).
+
 =item "digest-props" (B<OSSL_ASYM_CIPHER_PARAM_OAEP_DIGEST_PROPS>) <UTF8 string>
 
 Gets or sets the properties to use when fetching the OAEP digest algorithm.
 
+=item "digest-props" (B<OSSL_ASYM_CIPHER_PARAM_DIGEST_PROPS>) <UTF8 string>
```

Comment:
Duplicate entry?

Reply:
Well, yes and no. The above entry is specific to OAEP, and this is more generic - but they share the "digest-props" name. I decided to do it this way since they should never be used at the same time. An alternative approach would be to get rid of the OAEP specific version, and just make it use the generic parameter name.